server/services/config/crm5DisplayConfig.json

```diff
@@ -511,30 +511,30 @@
       "title": "Costs",
       "subsections": [
```

Comment:
Other disbursement field is missing for 'Costs accrued to Date' and 'Anticipated Costs'

Reply:
Other disbursement is present when there is api data. check 5001716